File: NativeIOS/NativeIOS/ViewController.m
//
//  ViewController.m
//  NativeIOS
//
//  Created by Malav Soni on 12/07/19.
//  Copyright © 2019 Malav Soni. All rights reserved.
//

#import "ViewController.h"
#import "AppDelegate.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
}
- (IBAction)showUnity_Clicked:(id)sender {
    [((AppDelegate *)[[UIApplication sharedApplication] delegate]) initUnity];
    [((AppDelegate *)[[UIApplication sharedApplication] delegate]) ShowMainView];

}


@end

File: SwiftNative/SwiftNative/ViewController.swift
//
//  ViewController.swift
//  SwiftNative
//
//  Created by Malav Soni on 23/07/19.
//  Copyright © 2019 Malav Soni. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
    }


    @IBAction func btnClicked(_ sender: Any) {
        (UIApplication.shared.delegate as! AppDelegate).initAndShowUnity()
    }
}

